DEVELOPMENT.md

```diff
@@ -27,9 +27,23 @@ npm install
 npm run setup
 ```
 
+### Auth0 tenant set up
+
+JunctionApp uses Auth0 for user authentication and authorization. To set up Auth0 for local development:
+
+1. Create an Auth0 account
+2. Follow [this](https://auth0.com/docs/deploy/deploy-cli-tool/create-and-configure-the-deploy-cli-application) tutorial to create a `auth0-deploy-cli-extension` in Auth0
+3. Add Auth0 Authorization extension to your tenant.
+4. Go to the Authorization extensions settings and enable API access
```

Comment:
"Auth0 Authorization" extension? 
![image](https://user-images.githubusercontent.com/1903693/121153778-07935600-c84f-11eb-80b1-447a71449b28.png)

If it's the right one, needs guidance how to install